app/scripts/modules/core/src/insight/insightLayout.component.html

```diff
@@ -3,6 +3,7 @@
   ng-if="!$ctrl.app.notFound && !$ctrl.app.hasError"
   ng-class="$ctrl.insightFilterStateModel.filtersExpanded ? 'filters-expanded' : 'filters-collapsed'"
 >
+  <filter-collapse ng-if="!$ctrl.insightFilterStateModel.filtersHidden" class="filter-toggle"></filter-collapse>
```

Comment:
I chose to directly place the component here rather than use the ui-view. This breaks the current pattern, but I feel it is much less complex.  The `ng-if` takes care of the conditional rendering for all dataSources. Directly placing the component here is much less complex than adding the new view to every `states` file and having to trace the DOM in a disjointed way.